Makefile

```
@@ -64,7 +64,7 @@ N_THREADS ?= $(shell nproc)
 
 #### Tools ####
 ifneq ($(shell type $(MIPS_BINUTILS_PREFIX)ld >/dev/null 2>/dev/null; echo $$?), 0)
-  $(error Please install or build $(MIPS_BINUTILS_PREFIX))
+  $(error Please install or build mips-linux-gnu [Unable to find $(MIPS_BINUTILS_PREFIX)ld])
```

I think having the error first is better, and would be more consistent overall with other things.

Additionally I tried expanding on suggested solutions without being too verbose but idk:

```suggestion
  $(error Unable to find $(MIPS_BINUTILS_PREFIX)ld. Please install or build MIPS binutils, commonly mips-linux-gnu. (or set MIPS_BINUTILS_PREFIX if your MIPS binutils install uses another prefix))
```